### PR TITLE
show feedback when peer is muted

### PR DESCRIPTION
**Pull request by gabrielsaruhashi**
image

### CODE REVIEW

File: src/components/Inbox/VoiceCall/VoiceCallInterface.tsx [hunk 1]
```diff
@@ -118,15 +121,20 @@ const VoiceCallInterface = ({ vc, authInfo, onTerminate }: Props) => {
         size={80}
         className="justify-center"
       />
+      <div className="flex gap-2 justify-center items-center">
+        <Typography.Text type="secondary" className="text-center">
+          {getFullNames(vc.visitors)}
+        </Typography.Text>
+        {!isPeerMuted && <AudioMutedOutlined className="text-red-500" />}
```
Comment (cainlevy) [inline src/components/Inbox/VoiceCall/VoiceCallInterface.tsx:128]: something looks odd here. does this show the muted icon when they are _not_ muted?

File: src/components/Inbox/VoiceCall/VoiceCallInterface.tsx [hunk 2]
```diff
@@ -105,9 +105,12 @@ const VoiceCallInterface = ({ vc, authInfo, onTerminate }: Props) => {
     }
   );
   const peer = peers[0];
+  const remoteAudioTracks = peer?.stream.getAudioTracks().length;
+  const isPeerMuted =
+    remoteAudioTracks === undefined || remoteAudioTracks > 0 ? false : true;
```
Comment (cainlevy) [inline src/components/Inbox/VoiceCall/VoiceCallInterface.tsx:110]: the undefined check here means they haven't joined yet?

**Review by cainlevy — APPROVED** [review]

File: src/components/Inbox/VoiceCall/VoiceCallInterface.tsx (hunk 2 above)
Comment (gabrielsaruhashi) [reply src/components/Inbox/VoiceCall/VoiceCallInterface.tsx:110]: yep!

File: src/components/Inbox/VoiceCall/VoiceCallInterface.tsx (hunk 1 above)
Comment (gabrielsaruhashi) [reply src/components/Inbox/VoiceCall/VoiceCallInterface.tsx:128]: oop! i changed this to record the demo :x and forgot to switch back

**Pull request merged**